tests/Resolver/ResolverTest.php

```diff
@@ -173,32 +173,32 @@ public function resolveWithNoAnswersShouldCallErrbackIfGiven()
 
     public function provideRcodeErrors()
     {
-        return array(
-            array(
+        return [
+            [
                 Message::RCODE_FORMAT_ERROR,
                 'DNS query for example.com (A) returned an error response (Format Error)',
-            ),
-            array(
+            ],
+            [
                 Message::RCODE_SERVER_FAILURE,
                 'DNS query for example.com (A) returned an error response (Server Failure)',
-            ),
-            array(
+            ],
+            [
                 Message::RCODE_NAME_ERROR,
                 'DNS query for example.com (A) returned an error response (Non-Existent Domain / NXDOMAIN)'
-            ),
-            array(
+            ],
+            [
                 Message::RCODE_NOT_IMPLEMENTED,
                 'DNS query for example.com (A) returned an error response (Not Implemented)'
-            ),
-            array(
+            ],
+            [
                 Message::RCODE_REFUSED,
                 'DNS query for example.com (A) returned an error response (Refused)'
-            ),
-            array(
+            ],
+            [
                 99,
                 'DNS query for example.com (A) returned an error response (Unknown error response code 99)'
-            )
-        );
+            ]
+        ];
```

Comment:
Same as `ParserTest`, sadly GitHub won't let me suggest the changes due to deleted lines.

Reply:
Good catch, adjusted as suggested :+1: